Loop.xcodeproj/project.pbxproj

```diff
@@ -4832,6 +4921,7 @@
 				INFOPLIST_FILE = "Loop Widget Extension/Bootstrap/Info.plist";
 				INFOPLIST_KEY_CFBundleDisplayName = "Loop Widgets";
 				INFOPLIST_KEY_NSHumanReadableCopyright = "Copyright © 2022 LoopKit Authors. All rights reserved.";
+				IPHONEOS_DEPLOYMENT_TARGET = 16.2;
```

Comment:
Please remove this change (found 8 times) from the pbxproj file.
Instead update `Loop.xcconfig` and change this section:

from:
```
// Min iOS Version [DEFAULT]                                    
IPHONEOS_DEPLOYMENT_TARGET = 15.1
```

to:
```
// Min iOS Version [DEFAULT]                                    
IPHONEOS_DEPLOYMENT_TARGET = 16.2
```